pallets/kensetsu/src/lib.rs

```diff
@@ -71,6 +71,16 @@ const VALIDATION_ERROR_CDP_SAFE: u8 = 4;
 /// Liquidation limit reached
 const VALIDATION_ERROR_LIQUIDATION_LIMIT: u8 = 5;
 
+#[derive(
+    Debug, Clone, Encode, Decode, PartialEq, Eq, PartialOrd, Ord, Copy, scale_info::TypeInfo,
+)]
+pub enum CdpType {
+    /// Pays stability fee in underlying collateral, cannot be liquidated.
+    V1,
+    /// Pays stability fee in stable coins, can be liquidated.
+    V2,
```

Comment:
Probably you can define more descriptive names

Reply:
Yeah, it from [the proposal](https://github.com/sora-xor/rfps/issues/60#issuecomment-1913677513).